ppr-ui/src/components/unitNotes/UnitNoteContentInfo.vue

```diff
@@ -12,14 +12,17 @@
       </v-col>
     </v-row>
 
-    <v-row v-if="note.expiryDateTime" no-gutters class="my-7">
+    <v-row v-if="isNoticeOfCautionOrRelatedDocType(note)" no-gutters class="my-7">
```

Comment:
This variable name is a bit too short, could we make it longer please?

Reply:
Would `isNoticeOfCautionOrRelatedDocumentTypeLikeContinuedNoticeOfCautionOrExtendedNoticeOfCautionDocumentTypes`

work?

Reply:
Seems much more appropriate YES!